### PR TITLE
NXP: Enable RTC counter support for the FRDM-K64F

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Enable the RTC counter present in the NXP K6x SoC series and list it as supported in the frdm_k64f board documentation.

### CODE REVIEW

File: dts/arm/nxp/nxp_k6x.dtsi [hunk 1]
```diff
@@ -41,6 +41,7 @@
 		uart-5 = &uart5;
 		usbd-0 = &usbd;
 		watchdog0 = &wdog;
+		rtc-0 = &rtc0;
```
Comment (MaureenHelm) [inline dts/arm/nxp/nxp_k6x.dtsi]: please move this up in the list to preserve alpha order

File: soc/arm/nxp_kinetis/ke1xf/Kconfig.series [hunk 2]
```diff
@@ -24,5 +24,6 @@ config SOC_SERIES_KINETIS_KE1XF
 	select HAS_MCUX_SCG
 	select HAS_MCUX_WDOG32
 	select HAS_MCUX_FTM
+	select HAS_MCUX_RTC
```
Comment (MaureenHelm) [inline soc/arm/nxp_kinetis/ke1xf/Kconfig.series]: it looks like this change is in the wrong file (ke1xf)?

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: soc/arm/nxp_kinetis/ke1xf/Kconfig.series (hunk 2 above)
Comment (henrikbrixandersen) [reply soc/arm/nxp_kinetis/ke1xf/Kconfig.series]: @MaureenHelm Thank you for reviewing. Sorry about that. My fingers are so used to typing "ke1x" ;-)

Fixed.

File: dts/arm/nxp/nxp_k6x.dtsi (hunk 1 above)
Comment (henrikbrixandersen) [reply dts/arm/nxp/nxp_k6x.dtsi]: Fixed.

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**